monai/transforms/compose.py

```diff
@@ -241,25 +233,27 @@ def __len__(self):
         """Return number of transformations."""
         return len(self.flatten().transforms)
 
-    def lazy_config(self):
-        """Return the lazy config to be passed to eval_lazy_stack."""
-        return {
-            "lazy_evaluation": self.lazy_evaluation,
-            "mode": self.mode,
-            "padding_mode": self.padding_mode,
-            "keys": self.lazy_keys,
-            "dtype": self.lazy_dtype,
-            "device": self.lazy_device,
-        }
-
-    def eval_lazy_stack(self, input_, upcoming_xform):
-        return _eval_lazy_stack(input_, upcoming_xform, **self.lazy_config())
+    def evaluate_with_overrides(self, input_, upcoming_xform):
+        """
+        Args:
+            input_: input data to be transformed.
+            upcoming_xform: a transform used to determine whether to evaluate with override
```

Comment:
this method looks strange, it takes a `upcoming_xform` as input but it's not really applying it cc @atbenmurray